lua/typescript-tools/protocol/text_document/code_action/batch/init.lua

```diff
@@ -54,15 +54,13 @@ function M.handler(request, response, params, ctx)
 
   local diagnostics = get_diagnostics_to_fix(params.diagnostics, params.error_codes)
 
-  local seqs = vim.tbl_map(function(diagnostic)
+  ctx.dependent_seq = vim.tbl_map(function(diagnostic)
```

Comment:
@KostkaBrukowa I don't have good idea for this property, maybe you have some better name?